content/dev/code/admin_sanitization.md

```diff
@@ -143,7 +143,7 @@ Zen Cart defines the following default case-insensitive sanitizers:
 
     POST values only 
     
-    uses ``[^0-9a-z.!@#$%^& ()`_+-~/\\]`` regex
+    uses ``[^0-9a-z.!@#$%^&()_-~`+^ \\]`` regex
```

Comment:
As commented in the prior PR, why does this have double back-ticks around it?

Reply:
it's required to preserve the back-tick before the plus sign.